sdk/keyvault/azure_security_keyvault_secrets/README.md

```diff
@@ -1,24 +1,429 @@
-# Azure KeyVault Secrets Client
+# Azure Key Vault Secret client library for Rust
 
-The `azure_security_keyvault_secrets` crate.
-These crates follow the [Azure SDK Design Guidelines for Rust](https://azure.github.io/azure-sdk/rust_introduction.html).
+Azure Key Vault is a cloud service that provides a secure storage of secrets, such as passwords and database connection strings.
+
+The Azure Key Vault Secrets client library allows you to securely store and control the access to tokens, passwords, API keys, and other secrets. This library offers operations to create, retrieve, update, delete, purge, backup, restore, and list the secrets and its versions.
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [Product documentation]
 
 ## Getting started
 
+### Install the package
+
+Install the Azure Key Vault Secrets client library for Rust with [Cargo]:
```

Comment:
I recommend lower-casing "secrets" everywhere.
```suggestion
Install the Azure Key Vault secrets client library for Rust with [Cargo]:
```
Best to compare with [.NET's](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Secrets/README.md). That was the rubric when I lead the KV v-team.

Reply:
Done